skbio/sequence/_iupac_sequence.py

```diff
@@ -19,6 +19,13 @@
 from ._sequence import Sequence
 
 
+def _invert_case(array):
+    # ASCII is built such that the difference between uppercase and
+    # lowercase is the 6th bit.
+    array ^= 32
```

Comment:
It might be better to have a constant to represent the ascii case offset instead of the function. @jairideout thoughts?